Sources/TuistCore/Constants.swift

```diff
@@ -4,7 +4,7 @@ public class Constants {
     public static let versionFileName = ".tuist-version"
     public static let binFolderName = ".tuist-bin"
     public static let binName = "tuist"
-    public static let gitRepositorySSH = "git@github.com:tuist/tuist.git"
+    public static let gitRepositorySSH = "https://github.com/tuist/tuist.git"
```

Comment:
Now that it is not longer containing a SSH url, this variable might be refactored to avoid including SSH on its name. I think something as `gitRepositoryURL` or similar might make sense.

Reply:
You are right. I'll update it. Good catch!

Reply:
https://github.com/tuist/tuist/commit/765c7fbd69eacbf7f0c804026ab444798808c469